### ISSUE TITLE
image mipmap tests do not enable extensions

### ISSUE THREAD

**Issue by b-sumner** [KhronosGroup/OpenCL-CTS#598]
The clang OpenCL headers do not expose the image mipmap functions unless cl_khr_mipmap_image* are defined.  However the CTS generated kernel sources do not actually enable the extensions.  I assume they should be enabling the extensions before using them?

**Comment by kpet** [comment]
Yes, we've certainly made quite a few similar changes to the CTS for other extensions.
